invenio_communities/views/decorators.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+from functools import wraps
+from flask import g
+from invenio_communities.proxies import current_communities
+
+
+def service():
+    """Get the record service."""
+    return current_communities.communities_service
+
+
+def pass_community(f):
+    """Decorate to retrieve the community record using the community service.
+    """
+    @wraps(f)
+    def view(**kwargs):
+        pid_value = kwargs.get('pid_value')
+        record = service().read(
```

Comment:
Comment: If we call everything a record I think it will get quite confusing in the end:
```suggestion
        community = service().read(
```